lib/core/engine/command/measure.js

```diff
@@ -71,14 +71,22 @@ class Measure {
       await setOrangeBackground(this.browser.getDriver(), this.options);
     } else {
       await setOrangeBackground(this.browser.getDriver(), this.options);
+
+      // Give ffmpeg/video on phone time to settle
+      if (isAndroidConfigured(this.options)) {
+        await delay(ANDROID_DELAY_TIME);
+      } else {
+        await delay(400);
+      }
+
```

Comment:
What is the reasoning behind adding another delay before recording has started?

Reply:
When getting screen recording working on Android, I needed various delays.  I'll try to make sure this lands without these delays; we can add them as we gain testers on Android.